2048/2048/src/main/java/com/appcoins/eskills2048/model/RoomResponse.java

```diff
@@ -31,10 +39,14 @@ public RoomStatus getStatus() {
     return users;
   }
 
+  public List<User> getUsersSortedByScore() {
+    return sortUsers(users);
```

Comment:
you should't change the original users list, did you consider creating a copy of it?

Reply:
agreed, changed